test/cli/suggest-sig/suggest-sig.out

```diff
@@ -447,3 +456,8 @@ class Abstract
   sig {abstract.params(a: T.untyped).void}
   def abstract_foo(a); end
 end
+
+sig {params(blk: T.untyped).returns(Integer)}
+def explicitly_named_block_parameter(&blk)
+  42
+end
```

Comment:
This is the new test. The other changes are fallout of the change to block argumentName printing.